src/counts.jl

```diff
@@ -231,22 +231,30 @@ end
 
 
 """
-    addcounts!(dict, x[, wv])
+    addcounts!(dict, x[, wv]; alg = :auto)
 
 Add counts based on `x` to a count map. New entries will be added if new values come up.
 If a weighting vector `wv` is specified, the sum of the weights is used rather than the
 raw counts.
+
+alg can be one of
+    :auto       -   default; uses :radixsort if safe to do so
+    :radixsort  -   if radixsort_safe(eltype(x)) == true then use radixsort to
+                    sort the input array in order to achieve faster performance
+                    by using more RAM. One should choose :dict if the
```

Comment:
I think the last sentence is redundant with the previous one. BTW, "faster performance" is not completely correct: code can be fast, but performance can be high, so use "higher performance" or something like that.

Reply:
I think it's easier for beginner to spell everything out even if it's obvious. Because when you are learning something there is a lot of information to process so sometimes these obvious things get missed.